### PR TITLE
Release/6.0.0

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Merge release branch into dev, getting ready to publish. 

### Added

-   Added new property `unitSpace` to `<pxb-channel-value>` and `<pxb-hero>` to manage spacing between the value and units.

### Changed

-   Changed the `<pxb-user-menu>` component to use a new `UserMenuHeaderComponent` in the menu overlay instead of using a `DrawerHeaderComponent`. 

### Fixed

-   Fixed stepper spacing in `<pxb-mobile-stepper>` when Back and Next buttons are uneven width.  
-   Fixed bug in `<pxb-app-bar>` that prevented class overrides on the root element.
-   Fixed bug in `<pxb-app-bar>` so scroll listeners can no longer attempt to measure `undefined` elements' scroll distance.

### CODE REVIEW

**Comment by emclaug2** [comment]
Can be viewed here:  https://pxb-storybook-ng-feature.web.app/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Change Log
 
+## v6.0.0 (Unreleased)
```
Comment (joebochill) [inline CHANGELOG.md:3]: might as well set the date since this is ready to go

File: components/src/core/channel-value/channel-value.component.ts [hunk 2]
```diff
@@ -37,8 +56,29 @@ export class ChannelValueComponent implements OnChanges {
...
+     * prefixUnitWhitelist: ['$']
+     *
+     * suffixUnitWhitelist: ['%', '℉','°F','℃','°C','°']
+     *
+     * @default auto;
+     * */
+    @Input() unitSpace: UnitSpaceType = 'auto';
 
+    prefixUnitWhitelist = ['$'];
+    suffixUnitWhitelist = ['%', '℉', '°F', '℃', '°C', '°'];
```
Comment (joebochill) [inline components/src/core/channel-value/channel-value.component.ts:76]: did we want to put '/100' in the suffix white list too?

**Review by joebochill — APPROVED** [review]
Code looks good to me

File: components/src/core/channel-value/channel-value.component.ts (hunk 2 above)
Comment (joebochill) [reply components/src/core/channel-value/channel-value.component.ts:76]: At this point, I would just log an enhancement for it if we want to

**Review by jeffvg — APPROVED** [review]
a lot of good changes here

**Comment by huayunh** [comment]
Plugged the dist into design patterns and everything worked :D

**Pull request merged**